react-app/.env.production

```diff
@@ -0,0 +1 @@
+REACT_APP_API_HOST=https://nickit.chalmers.it
```

Comment:
How come this doesn't have a port specified?

Reply:
If you don't specify the port it will use the default port for that protocol. 
(Which is `80` for `http` and `443` for `https`)

Reply:
I see!